README.md

```diff
@@ -68,7 +85,9 @@ docker build -t firecloud-orch .
 Run the container:
 ```
 docker run --name orch \
+  -p 8080:8080
```

Comment:
It didn't work for me empirically. I am also not particularly versed in docker, so it could be a problem in my env, too.